_posts/2019-12-20-differential-privacy/differential_privacy.Rmd

```diff
@@ -0,0 +1,470 @@
+---
+title: "Differential Privacy with TensorFlow"
+description: > 
+ Differential Privacy guarantees that results obtained from querying a database are basically independent of the presence in the data of a single individual. Applied to machine learning, we expect that no single training example influences the parameters of the trained model in a substantial way. This post introduces TensorFlow Privacy, a library built on top of TensorFlow, that can be used to train differentially private deep learning models from R.
+ 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019diffpriv
+bibliography: bibliography.bib
+date: 12-20-2019
+categories:
+  - Security
+  - Keras
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/cat.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+
+What could be treacherous about summary statistics? 
+
+The famous _cat overweight_ study (X. et al., 2019) showed that as of May 1st, 2019, 32 of 101 domestic cats held in Y., a cozy Bavarian village, were overweight. Even though I'd be curious to determine if my aunt G.'s cat (a happy resident of that village) has been fed too many treats and has accumulated some excess pounds, from the results reported in that study I can't know.
```

Comment:
> from the results reported in that study I can't know.

Maybe flip this line? "I can't determine that based on the results of the study."
(And then I suggest you change the earlier determine to "know" so it's not repetitive—also, I like having the informal language with your curiosity, and the more formal with the part referencing the study. 🙂 )

Reply:
makes sense! will change

Reply:
actually, found  yet sth else: "the study results don't tell."